tests/src/performance/Scenario/JitBench/Utilities/ProcessRunner.cs

```diff
@@ -67,7 +67,7 @@ public ProcessRunner(string exePath, string arguments, string replayCommand = nu
                 _p.StartInfo = psi;
                 _p.EnableRaisingEvents = false;
                 _loggers = new List<IProcessLogger>();
-                _timeout = TimeSpan.FromMinutes(10);
+                _timeout = TimeSpan.FromMinutes(60);
```

Comment:
good idea! 👍

Reply:
This changes the default timeout for every process JitBench launches. If you just have one particular process that is long running I'd suggest changing just that one rather than giving every process a free pass to be slow:

     new ProcessRunner("bla", "blabla").WithTimeout(TimeSpan.FromMinutes(60))